advanced/Scripts/webpage.sh

```diff
@@ -456,6 +460,7 @@ main() {
 		"setdns"            ) SetDNSServers;;
 		"setexcludedomains" ) SetExcludeDomains;;
 		"setexcludeclients" ) SetExcludeClients;;
+		"halt"              ) Halt;;
```

Comment:
True, but I avoided "poweroff" since it's kind of misleading on the Pi, since it doesn't actually power the device off, but just complete the halt action.  (Of course, this would likely behave differently on different hardware.)

Your call...  I'm fine with whatever you decide.